examples/jsm/controls/ArcballControls.js

```diff
@@ -67,7 +67,9 @@ const _endEvent = { type: 'end' };
 const _raycaster = new Raycaster();
 const _offset = new Vector3();
 
-
+const gizmoMatrixStateTemp = new Matrix4();
```

Comment:
Would you mind using this pattern?
```js
const _gizmoMatrixStateTemp = new Matrix4();
```
Module scope variable should use the `_` prefix.